hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/store/protocol/impl/pb/FederationProtocolPBTranslator.java

```diff
@@ -54,7 +54,7 @@ public FederationProtocolPBTranslator(Class<P> protoType) {
    * the proto handler this translator holds.
    */
   @SuppressWarnings("unchecked")
-  public void setProto(Message p) {
+  public void setProto(Message p) throws IllegalArgumentException {  // unchecked exception
```

Comment:
I think it is not necessary to add the comment `// unchecked exception` here.

Reply:
@xiaojunxiang2023 can you address this? we can commit post this

Reply:
I changed it myself, will merge if the build comes green